src/UserModE131.h

```diff
@@ -0,0 +1,67 @@
+#include <ESPAsyncE131.h>
+
+
+
+class UserModE131:public Module {
+
+public:
+
+  UserModE131() :Module("e131/sACN support") {
+    print->print("%s %s\n", __PRETTY_FUNCTION__, name);
+
+    print->print("%s %s %s\n", __PRETTY_FUNCTION__, name, success?"success":"failed");
+  };
+
+  //setup filesystem
+  void setup() {
+    Module::setup();
+    print->print("%s %s\n", __PRETTY_FUNCTION__, name);
+
+    if(!net->interfacesInited) {
+      print->print("No network yet, can't start e131");
+    }
+
+//    print->print("%s %s %s\n", __PRETTY_FUNCTION__, name, success?"success":"failed");
+  }
+
+  void loop(){
+    // Module::loop();
+    if(!net->interfacesInited) {
+      return;
+    }
+    if(!e131Created) {
+      print->print("Network exists, now init e131\n");
+      e131 = ESPAsyncE131(universeCount);
+      if (this->e131.begin(E131_MULTICAST, universe, universeCount)) { // TODO: multicast igmp failing, so only works with unicast currently
+        print->print("Network exists, begin e131.begin ok\n");
+        success = true;
+      }
+      else {
+          print->print("Network exists, begin e131.begin FALED\n");
+      }
+      e131Created = true;
+    }
+    if (!e131.isEmpty()) {
+        e131_packet_t packet;
+        e131.pull(&packet);     // Pull packet from ring buffer
+        
+        Serial.printf("Universe %u / %u Channels | Packet#: %u / Errors: %u / CH1: %u\n",
+                htons(packet.universe),                 // The Universe for this packet
+                htons(packet.property_value_count) - 1, // Start code is ignored, we're interested in dimmer data
+                e131.stats.num_packets,                 // Packet counter
+                e131.stats.packet_errors,               // Packet error counter
+                packet.property_values[1]);             // Dimmer data for Channel 1
+
+      bri = packet.property_values[1]; // TODO: ugly to just hack the global bri value
```

Comment:
Doesn't seem to take effect, but then also changing the compile-time default from 10 doesn't appear to change either, so I'm doing something wrong here

Reply:
<img width="622" alt="image" src="https://github.com/ewoudwijma/LedFix/assets/1737159/bdb8220d-4a43-4089-be4f-6269feb03548">
Because the actual brightness is changed in chFun of "bri", try calling mdl->setValue("bri", packet.property_values[1]);